ar6/mapping_template_AR6.csv

```diff
@@ -24,28 +24,29 @@ idx;Variable_AR6;Unit;r21m42;r21m42_unit;r21m42_factor;r21m42_spatial;m42;m42_un
 ;Carbon Sequestration|CCS|Fossil|Energy|Demand|Industry;Mt CO2/yr;;;;;;;;;;;;;;;;;
 27;Carbon Sequestration|CCS|Fossil|Energy|Supply;Mt CO2/yr;Emi|CO2|Carbon Capture and Storage|Fossil|Pe2Se;Mt CO2/yr;;;;;;Carbon Management|Carbon Capture|+|Fossil|Pe2Se;Mt CO2/yr;;;;;;;;total carbon dioxide emissions captured from fossil fuel use in energy supply (IPCC category 1A) and stored in geological deposits (e.g. in depleted oil and gas fields,unmined coal seams,saline aquifers) and the deep ocean,stored amounts should be reported as positive numbers
-28;Carbon Sequestration|CCS|Fossil|Energy|Supply|Electricity;Mt CO2/yr;;;;;;;;;;;;;;;;;total carbon dioxide emissions captured from fossil fuel use in electricity production (part of IPCC category 1A1a) and stored in geological deposits (e.g. in depleted oil and gas fields,unmined coal seams,saline aquifers) and the deep ocean,stored amounts should be reported as positive numbers
-29;Carbon Sequestration|CCS|Fossil|Energy|Supply|Gases;Mt CO2/yr;;;;;;;;;;;;;;;;;total carbon dioxide emissions captured from fossil fuel use in gaseous fuel production,excl. hydrogen (part of IPCC category 1A) and stored in geological deposits (e.g. in depleted oil and gas fields,unmined coal seams,saline aquifers) and the deep ocean,stored amounts should be reported as positive numbers
-30;Carbon Sequestration|CCS|Fossil|Energy|Supply|Hydrogen;Mt CO2/yr;;;;;;;;;;;;;;;;;total carbon dioxide emissions captured from fossil fuel use in hydrogen production (part of IPCC category 1A) and stored in geological deposits (e.g. in depleted oil and gas fields,unmined coal seams,saline aquifers) and the deep ocean,stored amounts should be reported as positive numbers
-31;Carbon Sequestration|CCS|Fossil|Energy|Supply|Liquids;Mt CO2/yr;;;;;;;;;;;;;;;;;total carbon dioxide emissions captured from fossil fuel use in liquid fuel production (part of IPCC category 1A) and stored in geological deposits (e.g. in depleted oil and gas fields,unmined coal seams,saline aquifers) and the deep ocean,stored amounts should be reported as positive numbers
+28;Carbon Sequestration|CCS|Fossil|Energy|Supply|Electricity;Mt CO2/yr;;;;;;;;Carbon Management|Carbon Capture|Fossil|Pe2Se|+|Electricity w/ couple prod;Mt CO2/yr;;;;;;;;total carbon dioxide emissions captured from fossil fuel use in electricity production (part of IPCC category 1A1a) and stored in geological deposits (e.g. in depleted oil and gas fields,unmined coal seams,saline aquifers) and the deep ocean,stored amounts should be reported as positive numbers
```

Comment:
same for all of these Carbon Management|Storage|...

Reply:
fixed